setup.cfg

```diff
@@ -146,6 +147,8 @@ build =
     black
     ruff
     pyqt5
+graph =
+    napari-graph
```

Comment:
Are there plans to push an early release of napari-graph to PyPI soon? It would make testing this branch out slightly easier and would need to be done before this is mergeable of course.

Reply:
uploaded